Added missing documentation in ABACAS and PlasmidID (#91)
nf-core/viralrecon: docs/output.md

  * `*.abacas.crunch`: Comparison file.
  * `*.abacas.fasta`: Ordered and orientated sequence file.
  * `*.abacas.gaps`: Gap information.

>   * `*.abacas.gaps.tab`: Gap information in tab format.

Reviewer: Oh, what is `tab` format? Do you mean tab-delimited format?

  * `*.abacas.MULTIFASTA.fa`: A list of ordered and orientated contigs in a multi-fasta format.
  * `*.abacas.tab`: Feature file
  * `*.unused_contigs.out`: Information on contigs that have a mapping information but could not be used in the ordering.

> * `assembly/<ASSEMBLER>/abacas/nucmer/`: Folder containing the files generated by the NUCmer algorithm used by ABACAS to

Reviewer: This sentence looks incomplete?

  * `images/<SAMPLE>_<REF_NAME>.png`: PNG file with the visualization of the alignment between the viral assembly and the reference viral genome.
  * `data/`: Files used for drawing the circos images.
  * `database/`: Annotation files used for drawing the circos images.

>   * `fasta_files`: Folder with fasta files. These fasta files correspond to the selection of those contigs/scaffolds that reconstruct the reference genome showed in images folder.

Reviewer: ```suggestion
  * `fasta_files`: Folder with fasta files that correspond to the selection of contigs/scaffolds required to reconstruct the reference genome generated in the `images/` folder.

```
